src/main/scala/tech/cryptonomic/conseil/routes/PlatformDiscovery.scala

```diff
@@ -21,36 +27,55 @@ object PlatformDiscovery {
   * @param config              configuration object
   * @param apiExecutionContext is used to call the async operations exposed by the api service
   */
-class PlatformDiscovery(config: PlatformsConfiguration)(implicit apiExecutionContext: ExecutionContext) extends LazyLogging with RouteHandling {
+class PlatformDiscovery(config: PlatformsConfiguration)(implicit apiExecutionContext: ExecutionContext, system: ActorSystem) extends LazyLogging with RouteHandling {
+
+  /** default caching settings*/
+  private val defaultCachingSettings: CachingSettings = CachingSettings(system)
+
+  /** simple partial function for filtering */
+  private val simpleKeyer: PartialFunction[RequestContext, Uri] = {
+    case r: RequestContext if r.request.method == HttpMethods.GET => r.request.uri
```

Comment:
is it necessary to check the method if the cache directive is applied downstream the `get` directive?

Reply:
not necessary - I used the one from example,

Reply:
I've removed the checking for `get`